.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -36,13 +36,18 @@
       - [3.1]
         - [ ] Verify that the `PrivateSourceBuiltArtifactsPackageVersion` in `eng/Versions.props` matches N-1 release artifacts.
         - [ ] Update the `global.json` SDK version to N-1: the latest released "3.1.XYY" SDK version, where "3.1.X" matches the SDK we're building.
-1.  - [ ] Regenerate patches as necessary.
-      - Run `./build.sh /t:CloneOnly /t:ApplyPatches` to minimally clone and attempt patch application to see what, if anything, needs to be regenerated.
-      - ASP.NET will always need to be regenerated because it contains a version number that needs to be updated.
 1.  - [ ] [2.1] Fetch internal git data for submodules.
       1.  Run `git submodule update --init --recursive` and see errors due to missing commits.
       1.  Run `fetch-vsts-commits.sh` with a PAT to automatically fetch internal refs.
+      1.  Run `git submodule update --init --recursive` again to update failed submodules.
           * If some refs are still missing on unusual repos, those repos may need to be set up in `fetch-vsts-commits.sh`.
```

Comment:
Maybe we should add a comment to `fetch-vsts-commits.sh` mentioning that there's no reason to "clean up" the list, in case that possibility occurs to someone again in the future?